src/test/rustdoc-js-std/path-ordering.js

```diff
@@ -0,0 +1,11 @@
+const QUERY = 'hashset::insert';
```

Comment:
Please add a comment explaining the goal of this test too.

Reply:
Oh, I liked the previous one since it confirmed the existence of the other results (from `HashMap` for example). Can you please put them back?